<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recuperação da senha</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 82, 208);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>560</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>560</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Verdana&quot;;</string>
       </property>
       <property name="text">
        <string>Recuperação da conta</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputEmailRec">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font: 8pt &quot;Verdana&quot; ;
	border: 2px solid rgb(130, 130,130);
	border-radius: 20px;
	padding: 15px;
	background-color: rgb(255, 255, 255);
	color: rgb(122, 82, 208);
}

QLineEdit:hover{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(122, 82, 208);
}

QLineEdit:focus{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(122, 82, 208);
}</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>E-MAIL</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonRec">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>330</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 8pt &quot;Verdana&quot; ;
	border: 2px solid rgb(122, 82, 208);
	border-radius: 5px;
	padding: 15px;
	color: rgb(122, 82, 208);
}

QPushButton:pressed{
	background-color: rgb(122, 82, 208);
	color: rgb(0, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>CONTINUAR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonBackRec">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 18pt &quot;Verdana&quot; ;
	border: rgb(255, 255, 255);
	color: rgb(122, 82, 208);
}

QPushButton:pressed{
	color: rgb(0, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelConfirEmail">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>155</y>
         <width>391</width>
         <height>41</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 200, 200);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
